apps/passport-client/src/dispatch.ts

```diff
@@ -444,6 +481,14 @@ function userInvalid(update: ZuUpdate) {
   });
 }
 
+function userHasChangedPassword(update: ZuUpdate) {
+  saveUserHasNewPassword(true);
+  update({
+    userHasChangedPassword: true,
+    modal: "another-device-changed-password"
+  });
+}
+
```

Comment:
`userHasNewPassword` could be named something that communicates what's going on better - eg. `changedPasswordOnOtherDevice`. the names are already long and verbose, might as well make em same the right thing